test/test-golden/nested-embed.adoc

```diff
@@ -101,7 +100,8 @@ Dictionary object with extension-specific objects.
 
 * **Type**: <<reference-extension,`extension`>>
 * **Required**: No
-* **Type of each property**: Extension
+* **Additional properties are allowed.**
+* **Type of each property**: `object`
```

Comment:
**tl;dr**: I think that the new output is indeed more correct and makes more sense. 

A quick side-by side of the old and the new state (from the markdown version) :

![Cesium wetzel properties](https://user-images.githubusercontent.com/5597569/167442290-a09615a9-0795-4fce-8ad9-8b0942d036a5.png)


---

Looking at the [code that is commented out after this change](https://github.com/CesiumGS/wetzel/pull/68/files#diff-38d2039530ae6b34c38c44810229b48bf851e5b7cd2a3c052dc4a865ebaacfdcR455) :  
```
if ((additionalProperties.type === 'object') && defined(schema.title)) {
    formattedType = style.linkType(schema.title, schema.title, autoLink);
}
```
This handles the case that there is no specific type information about the `additionalProperties` (i.e. it is only `object`). This could be the case for something like
```
...
"additionalProperties": {
    "type": "object",
    "title": "Additional property"
}
```
It might be that the intention originally was to have code like this:
```
if ((additionalProperties.type === 'object') && defined(additionalProperties.title)) {
    formattedType = style.linkType(additionalProperties.title, additionalProperties.title, autoLink);
}
```
to insert the _title of the additiona property type_ there (instead of the title of the containing schema)

Reply:
Yeah that makes a lot more sense. I think I might've tried it and got a bunch of unexpected failures, but would have to revisit to know for sure.

Reply:
Tried your variant. When parsing [image.schema.json]'s `metadata` property, the properties of which are [meta.schema.json](https://github.com/CesiumGS/wetzel/pull/68/files#diff-ffd4ed20cdbffc52e0bfb2cdfc64cb46fa7284a4dbb51815e65a24458993d71b), it emits:

``` md
### Image.metadata

* **Type**: `object`
* **Required**: No
* **Additional properties are allowed.**
* **Type of each property**: `meta`
```

Which seems odd to say the least, because the `meta.schema.json` schema does in fact have a title, `"title": "Metadatum", but it's just emitting the schema name, `meta`, instead. I guess that's valid as the _type_, but shouldn't it use the title?